FFmpeg-patches/0012_add_as10_shim_metadata.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/libavformat/bmxenc.c b/libavformat/bmxenc.c
+index 4a74462de4..da285c9cc9 100644
+--- a/libavformat/bmxenc.c
++++ b/libavformat/bmxenc.c
+@@ -154,6 +154,9 @@ static int bmx_write_header(AVFormatContext *s)
+     }
+ 
+     bmx_init(bmx->internal_data);
++    if (bmx->clip_type == CLIP_TYPE_AS10 && bmx->shim_name != 0) {
++        bmx_add_shim_metadata(bmx->internal_data, bmx->shim_name);
```

Comment:
This is a strange diff as you seems to introduce comments ?
It seems to be a dev comment.

Reply:
I don't understand, there is no comment here

Reply:
i am dumb 🐕